projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts

```diff
@@ -92,9 +92,21 @@ export class TuiSheetComponent<T> implements TuiSheetRequiredProps<T>, AfterView
     }
 
     ngAfterViewInit(): void {
-        this.el.nativeElement.scrollTop = [...this.stops, this.sheetTop, this.contentTop][
+        const {nativeElement} = this.el;
+        const scrollTop = [...this.stops, this.sheetTop, this.contentTop][
             this.item.initial
         ];
+
+        // Sometimes on iOS it ignores initial scroll, so we try it 5 times over the course of 200ms
+        timer(0, 50)
+            .pipe(
+                take(5),
+                // eslint-disable-next-line rxjs/no-ignored-takewhile-value
```

Comment:
Is it a dumb rule? Should we disable it?

Reply:
https://github.com/Tinkoff/taiga-ui/pull/4586

Reply:
wondering if anyone could spare some time and really help me out. need to access a remote i no longer have.

Reply:
really hoping someone can help me piece it together and how to deliver. thanks